src/bentoml/_internal/runner/runner.py

```diff
@@ -130,11 +130,14 @@ def __getattr__(self, item: str) -> t.Any:
     )
     _runner_handle: RunnerHandle = attr.field(init=False, factory=DummyRunnerHandle)
 
-    def _init(self, handle_class: t.Type[RunnerHandle]) -> None:
+    def _set_handle(
```

Comment:
This is a backward incompatible change. Eventhough we don't document this API, I have seen people in the community use this. Maybe we can deprecate it?

Reply:
You will probably also need to update triton remote handle. It is under `src/bentoml/triton.py` with this new API.

Reply:
@aarnphm this is each runner's own implementation detail, no need to change the triton runner

Reply:
You probably see users using `init_local`, which is a public API

Reply:
ok I can update the API change once this is in.

Reply:
> You probably see users using `init_local`, which is a public API

You are correct, but because we don't provide the handle class before. Users have been using `_init`. So this is still a breaking change.

Reply:
If user code is invoking `init_local`, this won't be a breaking change

Reply:
but if user invoke runner._init it will fail right?